modules/reference/pages/cluster-properties.adoc

```diff
@@ -1638,16 +1638,20 @@ Target replication factor for internal topics.
 
 ---
 
-=== minimum_topic_replications
+=== minimum_topic_replication
 
-Minimum permitted replication factor value for newly-created topics. Redpanda returns an error on any attempt to create a topic with a replication factor less than this property. When changing the `minimum_topic_replications` parameter, any previously existing topics having a replication factor less than the new value will continue to work as normal. Redpanda will, however, issue a warning level log message at start up indicating the existence of a topic with a replication factor less than the cluster's `minimum_topic_replications`.
+Minimum permitted replication factor value for newly-created topics. Redpanda returns an `INVALID_REPLICATION_FACTOR` error on any attempt to create a topic with a replication factor less than this property.
+
+When changing the `minimum_topic_replications` parameter, any previously existing topics having a replication factor less than the new value will continue to work as normal. Redpanda will, however, issue a warning level log message at start up indicating the existence of topics with a replication factor less than the cluster's `minimum_topic_replications`. That error message will read, `Topic X has a replication factor less than specified minimum: {} < {}`.
```

Comment:
```suggestion
When changing the `minimum_topic_replication` parameter, any previously existing topics having a replication factor less than the new value will continue to work as normal. Redpanda will, however, issue a warning level log message at start up indicating the existence of topics with a replication factor less than the cluster's `minimum_topic_replication`. That error message will read, `Topic X has a replication factor less than specified minimum: {} < {}`.
```

Reply:
Is there a better way of describing this warning message other than "{} < {}"?  Maybe just an example of "1 < 3" or something?

Reply:
Yeah, I'll just reword it as a representative example and put actual numbers in there. I also fixed those last minimum_topic_replications instances and made sure there weren't any more. Silly mistake to make, my apologies!